cmd/bridge/main.go

```
@@ -278,6 +278,10 @@ func run(ctx context.Context, args []string, stdout, stderr io.Writer) int {
 		return doctorCmd(args[1:], stdout, stderr)
 	case "update":
 		return updateCmd(ctx, args[1:], os.Stdin, stdout, stderr)
+	case "backup":
+		return backupCmd(args[1:], stdout, stderr)
```

golangci-lint (2.11.4)

[error] Function `backupCmd` should pass the context parameter

(contextcheck)